Issue 99 fix #99

revolunet opened the pull request:
ajout datasets + indexation CCNs

lionelB commented on packages/code-du-travail-frontend/src/sources.js:
```diff
@@ -1,24 +0,0 @@
-// mapping elastic search source type -> route name
```
Why did you remove it ?

revolunet replied:
argh PR splitting...

lionelB replied:
I see that in the PR "Unify Content Route" ;)

revolunet replied:
ok je remix :)

Commit:
feat(search): ajout conventions collectives fix #99 :

- ajout de la liste KALI et IDCC des conventions collectives dans ElasticSearch
- creation de pages vides avec redirection pour les conventions
